client/components/category-pill-navigation/style.scss

```diff
@@ -84,7 +84,7 @@
 			transform: rotate(0);
 		}
 
-		@media ( max-width: $break-wide ) {
+		@media ( max-width: $break-large ) {
```

Comment:
This works well, especially as touch devices like the iPad are able to swipe the pills. Is that the intention?